src/index.js

```diff
@@ -97,16 +97,16 @@ export {
 export { default as Navbar, NavbarBrand, NAVBAR_LABEL } from './Navbar';
 export { default as Overlay, OverlayTrigger } from './Overlay';
 export { default as PageBanner, PAGE_BANNER_DISMISS_ALT_TEXT } from './PageBanner';
+export { default as Pagination } from './Pagination';
 export {
-  default as Pagination,
   PAGINATION_BUTTON_LABEL_PREV,
   PAGINATION_BUTTON_ICON_BUTTON_NEXT_ALT,
   PAGINATION_BUTTON_ICON_BUTTON_PREV_ALT,
   PAGINATION_BUTTON_LABEL_PAGE_OF_COUNT,
   PAGINATION_BUTTON_LABEL_CURRENT_PAGE,
   PAGINATION_BUTTON_LABEL_NEXT,
   PAGINATION_BUTTON_LABEL_PAGE,
-} from './Pagination';
+} from './Pagination/constants';
```

Comment:
nit: i wonder if it's worth sticking to convention in this file of exporting constants from the top-level component file? Could consider adding the following to `./src/Pagination/index.jsx`:

```jsx
export * from './constants';
```

Reply:
Yeah, makes sense, updated, thanks!